### ISSUE TITLE
For `expand()`, `sizes` argument doesn't work while `size` argument works against the doc

### ISSUE THREAD

**Issue by hyperkai** [pytorch/pytorch#130960]
### 📚 The doc issue

[The doc](https://pytorch.org/docs/stable/generated/torch.Tensor.expand.html) of `expand()` says there is `sizes` argument as shown below:

> *sizes (torch.Size or [int](https://docs.python.org/3/library/functions.html#int)...) – the desired expanded size

But, setting `sizes` gets the error saying there is `size` argument as shown below:

```python
import torch

my_tensor = torch.tensor([7, 4, 5])

my_tensor.expand(sizes=(4, 3)) # Error
```

> TypeError: expand() missing 1 required positional arguments: "size"

So,  I set `size` argument following the error message above, then it works as shown below:

```python
import torch

my_tensor = torch.tensor([7, 4, 5])

my_tensor.expand(size=(4, 3))
# tensor([[7, 4, 5],
#         [7, 4, 5],
#         [7, 4, 5],
#         [7, 4, 5]])
```

### Suggest a potential alternative/fix

So, the doc should say `size` instead of `*sizes` and also should say `tuple` of `int` and `list` of `int` as shown below:

> *sizes (torch.Size, [int](https://docs.python.org/3/library/functions.html#int)..., `tuple` of `int` or `list` of `int`) – the desired expanded size

cc @albanD

**Comment by osbm** [comment]
Looking at the `Tensor.expand(*sizes)` line at the top of the page i would guess. That asterisk is meant as unpacking operator.

```

In [1]: x = torch.tensor([[1], [2], [3]])

In [2]: sizes = (3,4)

In [3]: x.expand(*sizes)
Out[3]:
tensor([[1, 1, 1, 1],
        [2, 2, 2, 2],
        [3, 3, 3, 3]])

In [4]: x.expand(3, 4)
Out[4]:
tensor([[1, 1, 1, 1],
        [2, 2, 2, 2],
        [3, 3, 3, 3]])
```

But i also agree, that page has very ambigious wording. It needs a clarification.

**Comment by malfet** [comment]
Yes, this is a common issue with our in-house argument parser, we do not always adhere to arg/kwarg pattern
